UnitsNet.Tests/CustomCode/ForceTests.cs

```diff
@@ -7,33 +7,36 @@ namespace UnitsNet.Tests
 {
     public class ForceTests : ForceTestsBase
     {
+        private const double StandardGravity = 9.80665;
+        private const double PoundsConversionRate = 4.4482216152605;
+
         protected override bool SupportsSIUnitSystem => true;
         protected override double DecanewtonsInOneNewton => 1E-1;
         protected override double DyneInOneNewton => 1E5;
 
-        protected override double KilogramsForceInOneNewton => 0.101972;
+        protected override double KilogramsForceInOneNewton => 1.0 / StandardGravity;
 
-        protected override double KilopoundsForceInOneNewton => 0.22481e-3;
+        protected override double KilopoundsForceInOneNewton => 1.0 / (PoundsConversionRate * 1_000.0);
         protected override double MeganewtonsInOneNewton => 1E-6;
         protected override double KilonewtonsInOneNewton => 1E-3;
 
-        protected override double KiloPondsInOneNewton => 0.101972;
+        protected override double KiloPondsInOneNewton => 1.0 / StandardGravity;
 
         protected override double NewtonsInOneNewton => 1;
 
         protected override double PoundalsInOneNewton => 7.23301;
 
-        protected override double PoundsForceInOneNewton => 0.22481;
+        protected override double PoundsForceInOneNewton => 1.0 / PoundsConversionRate;
 
-        protected override double TonnesForceInOneNewton => 1.019716212977928e-4;
+        protected override double TonnesForceInOneNewton => 1.0 / (StandardGravity * 1_000.0);
 
         protected override double MillinewtonsInOneNewton => 1.0e3;
 
         protected override double MicronewtonsInOneNewton => 1.0e6;
 
-        protected override double OunceForceInOneNewton => 3.596943089595368;
+        protected override double OunceForceInOneNewton => 1.0 / (PoundsConversionRate / 16.0);
 
-        protected override double ShortTonsForceInOneNewton => 1.12404471549816e-4;
+        protected override double ShortTonsForceInOneNewton => 1.0 / (PoundsConversionRate * 2_000.0);
```

Comment:
Please also change these test values to constants looked up in some reference source, so we don't repeat the same calculation as is performed in JSON and follow the current testing convention.

Reply:
I'm not sure I understand the request. Is there a similar solution you would like me to follow?

Reply:
The original value was `1.12404471549816e-4`. You can either revert back to this, or find a more accurate value in wikipedia, wolfram alpha, or some online converter. Similar for the other properties.

The purpose of these test values is to double check the conversion function in JSON and to be easily verifiable in some online converter.

Reply:
Sorry about that. I thought per your other requested change you wanted to hold off on making this change until #1309 was resolved.